turbinia/turbiniactl.py

```diff
@@ -620,7 +620,7 @@ def main():
     log.info('Available Jobs:')
     client.list_jobs()
   else:
-    log.warning('Command {0:s} not implemented.'.format(args.command))
+    log.warning('Command {!s} not implemented.'.format(args.command))
```

Comment:
I think this is what you want here:
```suggestion
    log.warning('Command {0!s} not implemented.'.format(args.command))
```

Reply:
Since it's only one argument, there is no need for 0. I can change the current format to stay consistent with the codebase.

Reply:
Yeah, let's change these to stay consistent.  Thanks!